projects/cashmere/src/lib/pipes/highlight/highlight-pipe.md

```diff
@@ -0,0 +1,50 @@
+The Highlight pipe is used to identify any text from the user's query.
+
+#### API
+
+```ts
+transform(value: string): string
+```
+
+`value`: the `string` value to query
+
+#### Importing the Highlight Pipe
+
+To use the `HighlightPipe`, import the `HighlightPipeModule` into your app's `CashmereModule` (which in turn is imported into your app):
+
+```ts
+import {NgModule} from '@angular/core';
+import {HighlightPipeModule} from '@healthcatalyst/cashmere';
+
+@NgModule({
+    exports: [HighlightPipeModule]
+})
+export class CashmereModule {}
+```
+
+Alternatively, each pipe in Cashmere is also exported from `PipesModule`, if you wish to import all pipes from the `@healthcatalyst/cashmere` package:
+
+```ts
+import {NgModule} from '@angular/core';
+import {PipesModule} from '@healthcatalyst/cashmere';
+
+@NgModule({
+    exports: [PipesModule]
+})
+export class CashmereModule {}
+```
+
+#### Using the Highlight Pipe in character mode
```

Comment:
I would change the header here to `Case Sensitivity`.  Character mode is actually one of the inputs to the ellipsis pipe so it doesn't apply here.  The description in this section is fine though